apps/meteor/ee/client/.eslintrc.json

```diff
@@ -0,0 +1 @@
+../../client/.eslintrc.json
```

Comment:
can we stop using symbolic links? they're way harder to keep, someone needs to remember they exists and need to be taken care of.

we could simply use `"extends": "../../client/.eslintrc.json"`